README.md

```diff
@@ -27,8 +20,53 @@ Fusebot was created for developers by developers at [fusebit.io](https://fusebit
 ## Installing Fusebot
 1. [Download Fusebot on Discord or Slack](https://fusebot.io?utm_source=github.com&utm_medium=referral&utm_campaign=fusebot-readme&utm_content=install)
 2. Run ```/fusebot coin DOGE``` to test command
-3. Run ```/fusebot edit coin``` to review code
-4. Run ```/fusebot help``` get started on your first command
+3. Run ```/fusebot edit coin``` and click the edit link to review how the code works
```

Comment:
I wonder if this is an unnecessary fork in the road. Removing it would help keep people on a "single track".

Reply:
Ok, yeah. I am removing it to keep them from going off on another path.